src/calculateRentalCost.js

```diff
@@ -28,7 +28,11 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalRentCost = days * 40;
+
+  return (days >= 7) ? totalRentCost - 50
+    : (days >= 3) ? totalRentCost - 20
+      : totalRentCost;
```

Comment:
ternary operator is not the best idea for this
Also one ternary inside another ternary makes code unreadable

It is better to use if conditions here